pkg/catalog/routes.go

```diff
@@ -22,6 +22,25 @@ const (
 	HTTPTraffic = "HTTPRouteGroup"
 )
 
+var allowAllRouteMatch trafficpolicy.HTTPRouteMatch = trafficpolicy.HTTPRouteMatch{
```

Comment:
nit : should we name this as `WildCardRouteMatch` ?

Reply:
updated to wildCardRouteMatch since we're only using this in this pkg for now